lib/fmrest/spyke/relation.rb

```diff
@@ -546,5 +577,27 @@ def with_clone
         end
       end
     end
+
+    class EmptyRelation
```

Comment:
Shouldn't this comply with the full public API of `Relation`?

Also, check my change involving `ImpossibleValue` above. I wonder if we couldn't use the presence of that "value" as a replacement for `EmptyRelation`. Let's discuss this.